www/jury/statistics.php

```diff
@@ -63,13 +63,24 @@
 // All problems
 $problems = $DB->q('SELECT p.probid,p.name FROM problem p
                     INNER JOIN contestproblem USING (probid)
-                    WHERE cid = %i ORDER by shortname', $cid);
+					WHERE cid = %i ORDER by shortname', $cid);
```

Comment:
Minor issue: whitespace changed into tabs.

Reply:
That was not the only part where indentation was mixed. Should be better now.